### ISSUE TITLE
[Feature Request] Multiple Garages w/ User Management

### ISSUE THREAD

**Issue by rtrickett337** [hargata/lubelog#136]
Combined with creating additional users/collaborators per vehicle, allow the ability to create multiple garages such that vehicles added to that garage inherit those permissions/user configurations. Use cases could include setting up additional garages for family members in other households to use LubeLogger who are less techy in setting this up (and wouldn't require spinning up another instance). If you have multiple garages/locations where vehicles are stored with different collaborators (perhaps Marine, Aviation, or additional homes) this could also be helpful in organizing your fleet within a single LubeLogger instance.

**Comment by nealhead** [comment]
Yes! I would love to be able to allow users to have their own garages.

**Comment by hargata** [comment]
Household feature will be available in the upcoming release, which allows you create households where each member of the household automatically has access to the vehicles in the household, allowing you to have multiple users with their own garages and households without having to spin up another instance.

See #1148

**Comment by hargata** [comment]
Household feature released in 1.5.5
